lib/screens/login/login2.dart

```diff
@@ -38,6 +40,12 @@ class _LoginFirebaseScreenState extends State<LoginFirebaseScreen> {
         .showSnackBar(new SnackBar(content: new Text(text)));
   }
 
+  void _toggleLoadingStatus() {
```

Comment:
Using a toggle for a loading screen is not a good idea as it makes a shimmer call relative to the previous state which may work but should not be the case. I would suggest sending the actual value inside the method instead of using `_isLoading = !_isLoading;`

Reply:
@im-Amitto I've changed this now. Do take a look.